script/generate-octicons.ts

```diff
@@ -24,12 +22,23 @@ const filePath = path.resolve(
   'sprite.octicons.svg'
 )
 
-const file = fs.readFileSync(filePath)
+const file = fs.readFileSync(filePath, 'utf-8')
+
+interface XML2JSResult {
```

Comment:
There's a tslint rule about interfaces starting with an `I` that is adding noise in here. Not sure whether this is also enforced by `eslint`, but we should add that back until we can do the transition.